### PR TITLE
Add cli option to set WM_CLASS property

### PR DESCRIPTION
**Pull request by akuntsch**
This makes it possible to run multiple instances of teams-for-linux with different task bar icons (by setting StartupWMClass in the .desktop file and the --class option to the same value).

### CODE REVIEW

File: app/config/index.js [hunk 1]
```diff
@@ -133,6 +133,11 @@ function extractYargConfig(configObject, appVersion) {
 				describe: 'Custom User Directory so that you can have multiple profiles',
 				type: 'string'
 			},
+			class: {
```
Comment (IsmaelMartinez) [inline app/config/index.js:136]: I think `class` is a bit too generic as a name. maybe `wmClass` or `windowManagerClass` ?

**Comment by IsmaelMartinez** [comment]
Those this fixes https://github.com/IsmaelMartinez/teams-for-linux/issues/438 ? Until now we have been telling people to use the customUserDir . See the discussion in https://github.com/IsmaelMartinez/teams-for-linux/discussions/764

**Comment by akuntsch** [comment]
> Thanks for the PR! Does this fixes #438 ? Until now we have been telling people to use the customUserDir . See the discussion in #764

The `--class` option goes in conjunction with `customerUserDir`. It simply makes it possible to have multiple instances of teams-for-linux running with different task bar icons (not system tray icons).

Without this PR, the icons of all running instances will stack in the task bar (at least in my KDE configuration), since they all use the same WM_CLASS property value.

Example of a `.desktop` file, that uses a custom icon and class:
```
[Desktop Entry]
Name=Teams for Linux Custom Icon
Exec=/opt/teams-for-linux/teams-for-linux --customUserDir=/home/user/.config/teams-for-linux-1 --class teams-for-linux-1 --awayOnSystemIdle --appIcon=/home/user/.local/share/icons/hicolor/512x512/apps/ms-teams-1.png %U
Terminal=false
Type=Application
Icon=ms-teams-1
StartupWMClass=teams-for-linux-1
Comment=Unofficial Microsoft Teams client for Linux using Electron. It uses the Web App and wraps it as a standalone application using Electron.
MimeType=x-scheme-handler/msteams;
Categories=Chat;Network;Office;
```
Note that the values of `--class` and `StartupWMClass` are the same, which makes the task bar icons work properly.

**Comment by IsmaelMartinez** [comment]
Lovely, thanks for contributing again. 

Would you be able to rename the config option to startupWMClass and increase the package.json version? Thanks!

**Comment by akuntsch** [comment]
I went with `--class` as that's kind of a convention, also used by Chrome for example. But I can change it, of course.

**Comment by IsmaelMartinez** [comment]
Fair enough. If you can pull the latest changes and add that config option in the https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/app/config/README.md that would be great (ideally alphabetically, so after chromeUserAgent)

Then, follow the (pre-)release process steps:
https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/CONTRIBUTING.md#release-process

It is basically updating 2 files. The release version can be 1.9.2 (if you make the change before another release). See image:

image

Once that is done, I am happy to pre-release it (I like to give it at least a day before promoting it to a release). 

Thanks again!

**Comment by akuntsch** [comment]
@IsmaelMartinez done :)

**Review by IsmaelMartinez — APPROVED** [review]

File: app/config/index.js (hunk 1 above)
Comment (IsmaelMartinez) [reply app/config/index.js:136]: this seems to be the standard name used by others,  so it is fine.

**Comment by IsmaelMartinez** [comment]
This has been pre-released as 1.9.2 . Thanks again for contributing!! https://github.com/IsmaelMartinez/teams-for-linux/releases/tag/v1.9.2

**Pull request merged**